File: js_basic_problems/04_count_even_numbers.js
// Problem 4: Count Even Numbers
// Input:
// 6
// 1 2 3 4 5 6
// Output:
// 3

const fs = require("fs");
const data = fs.readFileSync(0, "utf-8").trim().split("\n");
let idx = 0;
const next = () => data[idx++].trim();

let n = Number(next());
let arr = next().split(/\s+/).map(Number);
console.log(arr.filter(x => x % 2 === 0).length);

File: py_basic_problems/04_count_even_numbers.py
# Problem 4: Count Even Numbers
# Input:
# 6
# 1 2 3 4 5 6
# Output:
# 3

n = int(input())
arr = list(map(int, input().split()))
print(sum(1 for x in arr if x % 2 == 0))
